packages/web3-common/src/eth_execution_api.ts

```diff
@@ -168,14 +167,6 @@ export interface FeeHistoryResult {
 	readonly reward: number[][];
 }
 
-// https://github.com/ethereum/execution-apis/blob/main/src/schemas/filter.json#L28
-export interface Filter {
-	readonly fromBlock?: BlockNumberOrTag;
-	readonly toBlock?: BlockNumberOrTag;
-	readonly address?: Address | Address[];
-	readonly topics?: (Topic | Topic[] | null)[];
-}
-
```

Comment:
Moved to [web3-utils](https://github.com/ChainSafe/web3.js/blob/c7933b378db2e220ca0444f3481e1d0c4cb16d04/packages/web3-utils/src/types.ts#L103)